README.md

```diff
@@ -24,6 +24,57 @@ The server implements one tool:
   - Takes "name" and "content" as required string arguments
   - Updates server state and notifies clients of resource changes
 
+## Running the Project
```

Comment:
It looks like these are developer instructions. Let's make that clear and add separate instructions for users to run the project.

Reply:
changed